agent.go

```diff
@@ -118,15 +124,15 @@ func NewAgent(config Config) *Agent {
 
 // WrapHandler wraps the handler with the IOpipe agent
 func (a *Agent) WrapHandler(handler interface{}) interface{} {
-	logger.Debug(fmt.Sprintf("%s wrapped with IOpipe decorator", getFuncName(handler)))
+	a.log.Debug(fmt.Sprintf("%s wrapped with IOpipe decorator", getFuncName(handler)))
```

Comment:
I assume there is no way to add to the existing logger in Go to perform logging?